docs/languages/Java.md

```diff
@@ -28,7 +28,9 @@ TODO
 TODO
 
 ## Include Javadoc for properties
-TODO
+When you want to generate models from JavaDocs containing description and examples, you can use the `JAVA_DESCRIPTION_PRESET` option.
```

Comment:
It is not generating models from JavaDocs, but generating models which contain JavaDocs 🙂 The description is used throughout the PR that has to be adapted 🙂

Reply:
Sorry for this 😅 will fix this soon!

Reply:
No worries, take your time 👍 Thanks for solving 2 issues btw! 🎉